### DISCUSSION TITLE
Noob trying to get RTSP stream started with RPi 5 and RPi Camera Module 3 Wide

### DISCUSSION THREAD

**Discussion by GizmoGoody** [bluenviron/mediamtx#3942] [Questions]
### Question

The title says it all. The docker seems to be running:
![image](https://github.com/user-attachments/assets/f53b211c-5db8-4ffe-91f5-81e2040b4fa3)

But when I try the first command under [Basic Usage](https://github.com/bluenviron/mediamtx?tab=readme-ov-file#basic-usage) which is this...
`ffmpeg -re -stream_loop -1 -i file.ts -c copy -f rtsp rtsp://localhost:8554/mystream`
...I get this...
![image](https://github.com/user-attachments/assets/68a07e10-0413-4427-a9bb-69f7eea9a54e)

What do I do next?

**Comment by SMDIndomitable** [comment]
Hi, it means that is it not able to get the file.ts you're trying to stream. Perhaps provide an absolute path instead.

**Comment by GizmoGoody** [comment]
Thanks, but I don't get why the basic streaming example would reference a file. I want to view the real-time stream on VLC on my desktop for example. What's the right syntax to just start a stream of the camera without reference to a file?

**Comment by SMDIndomitable** [comment]
https://www.youtube.com/watch?v=rxtcyxV32nc&t=472s

**Comment by GizmoGoody** [comment]
Thanks! That video killed two birds...showed it wasn't just me who couldn't get the basic rpicam RTSP stream working using the provided docs and why I should just use MediaMTX (although those docs apparently are wrong too).
